src/main/java/gnu/io/CommPortIdentifier.java

```diff
@@ -163,7 +162,7 @@ public static void addPortName(String s, int type, CommDriver c)
 ------------------------------------------------------------------------------*/
 	private static void AddIdentifierToList( CommPortIdentifier cpi)
 	{
-		log.trace("CommPortIdentifier:AddIdentifierToList()");
+		log.trace("CommPortIdentifier:AddIdentifierToList: {}", cpi.getName());
```

Comment:
I think this can just come out altogether. I suspect there are really no trace-level logging statements we want to keep.